### PR TITLE
fix: assignment to function.call

### PR DESCRIPTION
**Pull request by Jack-Works**
## Description

image

Assign `call` property on a function thrown in our environment. I replaced that with `defineProperty`

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [x] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -534,3 +534,4 @@ Released with 1.0.0-beta.37 code base.
 ### Fixed
 -  Fix jsonrpc payload and response types (#4743) (#4761)
 -  Allowed more flexibility in typing the overly constrained `provider.disconnect` function (#4833)
+-  Fix web3-core-method throws on `f.call = this.call` when intrinsic is frozen (#4918)
```
Comment (jdevcs) [inline CHANGELOG.md:547]: This should be under 1.7.3

Comment (Jack-Works) [reply CHANGELOG.md:547]: thanks!

**Review by avkos — APPROVED** [review]

**Review by jdevcs — APPROVED** [review]

**Review by nazarhussain — APPROVED** [review]
Change seems good to me. But next time try to avoid including white space changes.

**Comment by Jack-Works** [comment]
> Change seems good to me. But next time try to avoid including white space changes.

thanks! they're automatically included by my IDE. I'll try to take care of that next time

**Pull request merged**